src/coreclr/System.Private.CoreLib/src/System/Runtime/InteropServices/ComWrappers.CoreCLR.cs

```diff
@@ -26,6 +26,20 @@ public static unsafe void GetIUnknownImpl(out IntPtr fpQueryInterface, out IntPt
         [SuppressGCTransition]
         private static partial void GetIUnknownImplInternal(out IntPtr fpQueryInterface, out IntPtr fpAddRef, out IntPtr fpRelease);
 
+        internal static unsafe void GetUntrackedIUnknownImpl(out delegate* unmanaged<IntPtr, uint> fpAddRef, out delegate* unmanaged<IntPtr, uint> fpRelease)
+        {
+            fpAddRef = GetUntrackedAddRef();
+            fpRelease = GetUntrackedRelease();
+        }
+
+        [LibraryImport(RuntimeHelpers.QCall, EntryPoint = "ComWrappers_GetUntrackedAddRef")]
+        [SuppressGCTransition]
+        private static unsafe partial delegate* unmanaged<IntPtr, uint> GetUntrackedAddRef();
```

Comment:
Should these be marked `Member` calling convention?

Reply:
I can mark them as `Member`. We haven't generally done that in this space, but it is more correct.

Reply:
From what I recall for `IUnknown`, we generally just use `IntPtr`. Do we need the function pointer signature at all in this case?